admin-ui/app/components/bootstrap/bootstrap.html

```diff
@@ -18,29 +18,29 @@
   <div ng-if="bootstrap.types.android">
     <div class="form-group">
       <label class="col-sm-3 control-label" for="fcmApiKey"></label>
-      <div class="col-sm-7">Server Key<br>
+      <div class="col-sm-7"><span translate>Server Key</span><br>
         <input type="text" placeholder="e.g. AIza5a448c2f31700a466fbc9d33d33942b043a27596" id="fcmApiKey" class="form-control" ng-model="bootstrap.config.android.googleKey" ng-required="bootstrap.types.android">
       </div>
     </div>
     <div class="form-group">
       <label class="col-sm-3 control-label" for="fcmProjectNumber"></label>
       <div class="col-sm-7">
-        Sender ID<br>
+        <span translate>Sender ID</span><br>
         <input type="text" placeholder="e.g. 42" id="fcmProjectNumber" class="form-control" ng-model="bootstrap.config.android.projectNumber" ng-required="bootstrap.types.android">
       </div>
     </div>
   </div>
 
 
   <div class="form-group" ng-if="bootstrap.allowCreate('ios')">
-    <label class="col-sm-3 control-label sr-only">iOS</label>
+    <label class="col-sm-3 control-label sr-only" translate>iOS</label>
```

Comment:
Language what I can see in the past that such proper noun is translated is Chinese though, I'm not sure if this is the case. I'm usually taking a conservative way and make all of strings localized unless it is a sort of enumerated value and affects its behavior in the code.
However, I don't have strong opinion on it and it works for me so far without localizing them. I withdraw them so far and we could add it anytime as needed.